qa-src/Controllers/User/UsersList.php

```diff
@@ -18,13 +18,25 @@
 
 namespace Q2A\Controllers\User;
 
+use Q2A\Auth\NoPermissionException;
+use Q2A\Middleware\Auth\InternalUsersOnly;
+use Q2A\Middleware\Auth\MinimumUserLevel;
+
 require_once QA_INCLUDE_DIR . 'db/users.php';
 require_once QA_INCLUDE_DIR . 'db/selects.php';
 require_once QA_INCLUDE_DIR . 'app/users.php';
 require_once QA_INCLUDE_DIR . 'app/format.php';
 
 class UsersList extends \Q2A\Controllers\BaseController
 {
+	public function __construct()
+	{
+		parent::__construct();
+
+		$this->addMiddleware(new InternalUsersOnly(), array('newest', 'special', 'blocked'));
+		$this->addMiddleware(new MinimumUserLevel(QA_USER_LEVEL_MODERATOR), array('blocked'));
```

Comment:
`[]`

Reply:
Remember this branch needs to be PHP 5.3 compatible